test/legacy_test/test_batch_norm_op.py

```diff
@@ -375,6 +375,77 @@ def test_check_output(self):
     def init_kernel_type(self):
         pass
 
+    def check_without_scale_and_bias(self, place, data_layout, dtype, shape):
```

Comment:
This function seems not be called since `test_check_output` still calls `check_with_place`? And locally tested, this function will report error for creation of op...

Reply:
You're right, I'll fix it.